heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -737,7 +747,9 @@ public void readData(long n) {
         public void retainData(final long n) {
             long curRetainedDataPoints = retained.addAndGet(n);
             if (curRetainedDataPoints > 500_000) {
-                log.info("Current retained data points num: {}", curRetainedDataPoints);
+                long total = quotaWatchers.stream().map(QuotaWatcher::getRetainData).reduce(0L, Long::sum);
+                log.info("Total: {}; watchers: {}; Data points retained: {}; added: {}", total,
```

Comment:
Can we distinguish the log.info messages a bit more? I have to read most of the line to distinguish between retained and read data. Also, let's spell-out the different numbers E.g.

```log
blah.info: Sep 3, 2020...: Data Points READ: Instance Total: x, This Query Total: y, Delta: z (# Watchers: q)
...
blah.info: Sep 3, 2020...: Data Points RETAINED: Instance Total: x, This Query Total: y, Delta: z (# Watchers: q)
```